dev-lang/crystal/crystal-0.7.7.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit multilib
+
+BV=${PV}-1
+BV_AMD64=${BV}-linux-x86_64
+BV_X86=${BV}-linux-i686
+
+DESCRIPTION="The Crystal Programming Language"
+HOMEPAGE="http://crystal-lang.org"
+SRC_URI="https://github.com/manastech/crystal/archive/${PV}.tar.gz -> ${P}.tar.gz
+	amd64? ( https://github.com/manastech/crystal/releases/download/${PV}/crystal-${BV_AMD64}.tar.gz )
+	x86? ( https://github.com/manastech/crystal/releases/download/${PV}/crystal-${BV_X86}.tar.gz )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc examples +xml +yaml"
+
+DEPEND="
+	dev-libs/boehm-gc[static-libs]
```

Comment:
@mgorny 
The following is the cyrstal issue about this problem:
https://github.com/manastech/crystal/issues/1382 
As the issue say, crystal use a function of bdw-gc(boehm-gc is gentoo portage) that is not exposed in the current release.
Maybe I can make a patch for dev-libs/boehm-gc to export the function in dynamic library, just like this: https://github.com/ivmai/bdwgc/commit/e2a14481796cd4b073298a9c895b03015c72e02c